_articles/code-of-conduct.md

```diff
@@ -100,6 +100,16 @@ Sometimes, a resolution cannot be reached. The person in question may become agg
 
 Banning members should not be taken lightly and represents a permanent and irreconcilable difference of perspectives. You should only take these measures when it is clear that a resolution cannot be reached.
 
+## Your responsibilities as a maintainer
+
+A code of conduct is not a law that is enforced arbitrarily. You are the enforcer of the code of conduct and it's your responsibility to follow the rules that the code of conduct establishes.
+
+As a maintainer you establish the guidelines for your community and enforce those guidelines according to the rules set forth in your code of conduct. This means taking any report of a code of conduct violation seriously. The reporter is owed a thorough and fair review of their complaint. If you determine that the behavior that they reported is not a violation, communicate that clearly to them and explain why you're not going to take action on it. What they do with that is up to them: tolerate the behavior that they had an issue with, or stop participating in the community.
+
+A report of behavior that doesn't *technically* violate the code of conduct may still indicate that there is a problem in your community, and you should investigate this potential problem and act accordingly. This may include revising your code of conduct to clarify acceptable behavior and/or talking to the person whose behavior was reported and telling them that while they did not violate the code of conduct, they are skirting the edge of what is expected and are making certain participants feel uncomfortable.
+
+In the end, as a maintainer, it's your community and you set and enforce the standards for acceptable behavior. You have the ability to shape your community values, and participants in your project will expect you to enforce those values in whatever way you see fit.
```

Comment:
> …in whatever way you see fit.

I could see how this could be misconstrued or taken out of context. Would it make sense to either leave this off, or change to something that suggests an expectation of responsibility as the first paragraph empahsizes?